src/hotspot/os/aix/os_perf_aix.cpp

```diff
@@ -22,519 +22,379 @@
  *
  */
 
-#include "precompiled.hpp"
+#include "utilities/debug.hpp"
 #include "jvm.h"
+#include "libperfstat_aix.hpp"
 #include "memory/allocation.inline.hpp"
+#include "memory/resourceArea.hpp"
 #include "os_aix.inline.hpp"
+#include "precompiled.hpp"
```

Comment:
precompiled.hpp must appear at the very top of the cpp file. The one exception from the "order includes alphabetically" rule. Oh, that, and people usually include system headers after hotspot ones.

Reply:
:+1: 

Also, since I'm thinking of it here. I ran without precompiled.hpp as you mentioned elsewhere. No issues detected.